data_transfer/config.py

```diff
@@ -15,6 +15,8 @@
 class Settings(BaseSettings):
     database_uri: str
 
+    dmp_study_id: str
+
```

Comment:
My thinking here was that the user of the dmpy client might want to create multiple objects that upload to multiple study sites, i.e., to allow flexability. However, this would require creating multiple `config.py` since one maps to a study ID.

I could instead move this to `dmpy.env` as it is currently only set once?

Reply:
Note: [moved this to dmpy.env](https://github.com/ideafast/dmpy/pull/2/commits/25f2432117348034147ab840a51b4d13035ddde5) and [removed it from this repo](https://github.com/ideafast/middleware-services/pull/44/commits/24c84318ac0d55d340ada86f47bc9b1b473a4f42)